Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -10,6 +10,7 @@ import dayjs from "dayjs";
 import { DatePicker } from "@mui/x-date-pickers/DatePicker";
 import { MobileTimePicker } from "@mui/x-date-pickers/MobileTimePicker";
 import Field from "../../../Components/Inputs/Field";
+import axios from "axios";
```

Comment:
We have a `NetworkService` class that provides an axios connection with the app base URL set up and methods for connecting to the backend.

We can remove this and import network service instead.  See other pages that make network requests for examples :+1: